numcodecs/blosc.c

```diff
@@ -304,7 +305,7 @@ END: Cython Metadata */
   #undef BASE
   #undef MASK
   #ifdef SIZEOF_VOID_P
-    enum { __pyx_check_sizeof_voidp = 1 / (int)(SIZEOF_VOID_P == sizeof(void*)) };
+    enum { __pyx_check_sizeof_voidp = 1/(SIZEOF_VOID_P == sizeof(void*)) };
```

Comment:
It seems `blosc` is changing here. Can we back out those changes as they are unrelated?

Reply:
This change is same as the current master. Do you want roll back?

Reply:
Ah didn't realize that. Sorry. Maybe we can merge the branch with `master` to clear this from the diff then?